encryption.rst

```diff
@@ -39,11 +39,10 @@ production use.
 
 .. note::
```

Comment:
We can remove this `note` section because it just talks about old behavior of singularity at a specific version